runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    Extra default keymaps                        {Nvim}
+For nvim.    Last change:      2015 May
+
+*nvim-extra-keymaps*
+
+The `plugin/extra-keymaps.vim` script provides a few extra keymaps that might
+be useful and tries to fix some issues the community has identified with vim's
+default keymaps.
+
+The user can disable the plugin by having >
+
+    let g:nvim#defaults#maps#use = 0
+
+in their |vimrc| (additionally, all features can be disabled separately,
+using the variables described below).
+
+============================================================================
+The keymaps provided are:
+
+                         *t_<Esc><Esc>*  *gr:nvim#defaults#maps#leave_term_mode*
+1. Leave |terminal-mode| using <esc><esc>, besides the usual  CTRL-\ CTRL-N
+   (see |c_CTRL_\_CTRL_N|).
+
+                                            *Y-eol* *g:nvim#defaults#maps#Y_eol*
+2. |Y| yanks to the end of line.
+
+                                *undo-ctrl-u* *g:nvim#defaults#maps#undo_ctrl_u*
+3. <C-u> can be undone. (see |i_CTRL-U|)
+
+			          *smart-home* *g:nvim#defaults#maps#smart_home*
```

Comment:
I can picture it so clearly: You have two foot pedals, one activates tabs, and one spaces, and you hit each one arbitrarily until your cursor is at the desired indentation level :grin: